.github/pull_request_template.md

```diff
@@ -3,5 +3,3 @@
 <!-- Please always prepend your PR title with tags such as [Metal], [CUDA], [Doc], [Example]. Use a lowercased tag (e.g. [cuda]), for PRs that are invisible to end-users (e.g. intermediate implementation). More details: http://taichi.readthedocs.io/en/latest/contributor_guide.html#pr-title-tags -->
 
 Related issue = #... (if any)
-
-[[Click here for the format server]](http://kun.csail.mit.edu:31415/)
```

Comment:
Maybe we should keep this here for a while, just in case some newcomers do not have access to reviewer assignment?

Reply:
Oh, sorry, I almost forget that review requesting needs write-access...